config/routes.rb

```diff
@@ -136,6 +141,11 @@
       get "/recruitment_cycles/:recruitment_cycle_year/providers/suggest", to: "providers#suggest"
 
       resources :organisations, only: :index
+      resources :accredited_body,
```

Comment:
Is this resource attached to a controller? And where is it being used?

Reply:
It is not.. Experiment hangover. Removed in 3c2e7a0